llvm/lib/Target/SPIRV/SPIRVGlobalRegistry.cpp

```diff
@@ -629,7 +629,7 @@ SPIRVGlobalRegistry::getOrCreateConstNullPtr(MachineIRBuilder &MIRBuilder,
   unsigned AddressSpace = typeToAddressSpace(LLVMTy);
   // Find a constant in DT or build a new one.
   Constant *CP = ConstantPointerNull::get(
-      PointerType::get(::getPointeeType(LLVMTy), AddressSpace));
+      PointerType::get(::getPointeeType(LLVMTy)->getContext(), AddressSpace));
```

Comment:
Shouldn't need to go through getPointeeType here.

Reply:
Okay cool, I wasn't too sure. I'll change it thanks!